docs/quickstart/index.rst

```diff
@@ -0,0 +1,301 @@
+.. _ksql_quickstart:
+
+Quick Start
+===========
+
+.. important::
+
+    This release is a developer preview. It is strongly recommended that you test before running KSQL against a production Kafka cluster.
+
+The goal of this quick start is to demonstrate a simple workflow using
+KSQL to write streaming queries against data in Kafka.
+
+Setup
+-----
+
+.. toctree::
+    :maxdepth: 3
+    :hidden:
+
+    quickstart-non-docker
+    quickstart-docker
+
+Because KSQL queries data in a Kafka cluster, you will need to bring up a Kafka cluster, including ZooKeeper and a Kafka broker.
+
+1. Bring up a Kafka cluster and start KSQL. Follow the instructions based on whether you are using Docker:
+
+    -  :ref:`ksql_quickstart_docker` (recommended)
+    -  :ref:`ksql_quickstart_non_docker`
+
+2. After you have successfully started the Kafka cluster and started
+   KSQL, you will see the KSQL prompt:
+
+   .. code:: bash
+
+                          ======================================
+                          =      _  __ _____  ____  _          =
+                          =     | |/ // ____|/ __ \| |         =
+                          =     | ' /| (___ | |  | | |         =
+                          =     |  <  \___ \| |  | | |         =
+                          =     | . \ ____) | |__| | |____     =
+                          =     |_|\_\_____/ \___\_\______|    =
+                          =                                    =
+                          =   Streaming SQL Engine for Kafka   =
+       Copyright 2017 Confluent Inc.                         
+
+       CLI v0.1, Server v0.1 located at http://localhost:9098
+
+       Having trouble? Type 'help' (case-insensitive) for a rundown of how things work!
+
+       ksql>
+
+3. KSQL provides a structured query language to query Kafka data, so you
+   need some data to query. For this quick start, you will produce mock
```

Comment:
We don't produce "mock" streams here IMHO.  We produce *real* streams, even though the data we write to the streams is fictitious/made up.  I think this is an important difference -- we are actually writing data to the Kafka cluster, it's not "mocked" in-memory similar to unit testing.